Tools/CISupport/ews-build/steps.py

```diff
@@ -543,9 +546,11 @@ def start(self):
         d.addCallback(lambda res: shell.ShellCommand.start(self))
 
     def hideStepIf(self, results, step):
-        return results == SUCCESS and self.getProperty('sensitive', False)
+        return not self.doStepIf(None) or (results == SUCCESS and self.getProperty('sensitive', False))
```

Comment:
Should it be self.doStepIf(None) or self.doStepIf(step)

Reply:
Didn't even notice we had the `step` variable there, definitely `self.doStepIf(step)`